rules/do-you-provide-the-reason-behind-the-rules-rather-than-just-enforce-them/rule.md

```diff
@@ -8,23 +8,66 @@ created: 2012-09-25T17:54:42.0000000Z
 authors:
 - title: Adam Cogan
   url: https://ssw.com.au/people/adam-cogan
+- title: Luke Cook
+  url: https://ssw.com.au/people/luke-cook
 related: []
 redirects:
 - general-standards-watchdog-do-you-provide-the-reason-behind-the-rules-rather-than-just-enforce-them
 
 ---
 
-If you criticize your staff:
+Learning lessons the hard way is a fact of life, but one of the great things about teamwork is that you can help others avoid making the same mistakes over and over again. This is the foundation of great standards! But what happens when a new (or old) team member misses a standard?
 
-* You don't make lasting changes
-* They will resent you, resulting in lower productivity and employee morale
+<!--endintro-->
 
+Everyone makes mistakes. If you run around wielding your authority as a cudgel, telling them they'd better comply *or else*, two things will happen:
 
-<!--endintro-->
+1. They will resent you
+2. They will only bother following standards when you're around
+
+For example: if one of your standards is for developers to send "test please" emails, there's a hard-learned lesson behind that standard (or else it wouldn't exist!). You can beat your developers over the head with the rule, or you can take 5 minutes to explain the reasons why it exists and why it's important.
+
+::: email-template
+|          |     |
+| -------- | --- |
+| From:    | Boss |
+| To:      | Mark |
+| Subject: | No test please email |
+::: email-content  
+
+### Hi Mark,
+
+You didn't send a "test please" email. Please do that next time.
+
+Boss
+
+:::
+:::
+::: bad
+Figure: Bad example - This email doesn't tell Mark why this is so important. 
+:::
+
+
+::: email-template
+|          |     |
+| -------- | --- |
+| From:    | Boss |
+| To:      | Mark |
+| Subject: | Test Please emails standard |
+::: email-content  
+
+### Hi Mark,
+
+Regarding that PBI you worked on yesterday.
 
-For example: If one of your responsibilities is to make sure all your developers sent "test please" emails, you can run around with a lot of authority telling them they'd better comply, but at soon as you're not around, they will resent you and not bother about it.
+We have a standard about sending Test Please emails to the client (check out [this rule](https://www.ssw.com.au/rules/request-a-test-please/)).
 
-A better approach is, when you see developers not sending "test please" emails, explain that they save time by getting early feedback, allowing bugs to be fixed while it's still fresh in the developers' minds. Explain to them that the cost of a bug increases over time.
+This saves time by getting early feedback, allowing bugs to be fixed while it's still fresh in the developer's mind.
 
+The longer the feedback loop takes, the more expensive a PBI becomes to the client :(
 
-![Figure: "The floggings will continue until morale improves"](DontCriticize.jpg)
```

Comment:
Even if the attempted humour is not super funny, I prefer to have it to keep things interesting. So I wouldn't want to kill this without something better.

Reply:
I disagree @adamcogan 
+1 to remove the unprofessional/low quality/cartoonish image

It's not just because it isn't "super funny"... the image is inconsistent with the sentence and with SSW style.

I would recommend **using the caption sentence as a quote** though:

> The floggings will continue until morale improves

Reply:
@tiagov8 and @adamcogan 

We could use the phrase directly after the intro as an opener:

> *"The floggings will continue until morale improves"*
> \- A bad manager

Learning lessons the hard way is a fact of life... [etc.]

Does that work?

Reply:
Works to me